docs/docs/guides/start/quickstart.mdx

```diff
@@ -103,49 +103,69 @@ The order of creation for the above components may vary depending on the specifi
 
 ![Login](/img/guides/quickstart/v3_8.png)
 
-### 2. Create your first instance
+### 2. Complete onboarding questions
 
-As a user of the ZITADEL Cloud Customer Portal, you now can create multiple instances to suit your specific needs. This includes instances for development, production, or user acceptance testing, as well as instances for different clients or applications. For example, you might create an instance for each product in a B2C scenario, or an instance for each tenant or customer in a B2B scenario. The possibilities are endless. You can create your first instance for free.
+To begin, we'll ask you a few questions.
+These will help us understand your needs and personalize your ZITADEL experience, ensuring you get the most out of it.
 
-1. Let’s create an instance. Click on “Create new instance”.
+![Onboarding Questions](/img/guides/quickstart/onboarding_questions.png)
 
-![2-factor Authentication](/img/guides/quickstart/v3_9.png)
+### 3. Create your first instance
+
+As a user of the ZITADEL Cloud Customer Portal, you now can create multiple instances to suit your specific needs.
+This includes instances for development, production, or user acceptance testing, as well as instances for different clients or applications.
+For example, you might create an instance for each product in a B2C scenario, or an instance for each tenant or customer in a B2B scenario.
```

Comment:
```suggestion
```

Reply:
I would leave this out here. Could create confusion. I guess the default for such scenarios would be to use organizations. Some cases might require instances, but I guess that's overload here and could irritate.